Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -65,6 +65,10 @@ def http_request(self, url_suffix: str, requests_kwargs) -> Dict:
             demisto.error(str(res.content))
             raise error
 
+    @property
+    def _headers(self):
+        return get_headers(self._params)
```

Comment:
add a comment in the code to explain the rationale of calling the function each time instead of once (why the original flow was flawed)